Cogs/events.py

```diff
@@ -49,7 +48,7 @@ async def clear_error(self, ctx, error):
             print(f"non-admin {ctx.message.author} tried to use createevent")
         else:
             await ctx.send(embed = discord.Embed(title = "Unknown error. Please contact developers to check logs", color = discord.Color.red()))
-            print("Create Event error: ", error, error.withtraceback())
+            raise Exception("Create Event error: ", error)
```

Comment:
```suggestion
            print("Create Event error: ", error)
            raise error
```

Reply:
Ohh ok, thought your comment in the issue post was just generic pseudocode not the actual code